### PR TITLE
Ignore gnu-config for KB-H014

### PR DESCRIPTION
**Pull request by uilianries**
Required by https://github.com/conan-io/conan-center-index/pull/2302

/cc @madebr

### CODE REVIEW

**Comment by madebr** [comment]
@uilianries 
Can `autoconf` and `automake` be added to the ignore list as well?
Because now, we manually add a `.exe` extension which is awkward, imho.

See https://github.com/conan-io/conan-center-index/pull/657/commits/deae1a38eb2ca161225572958cacbe10a7a491b4 for the patch for autoconf.

File: hooks/conan-center.py [hunk 1]
```diff
@@ -606,7 +606,7 @@ def test(out):
         if conanfile.version == "system":
             return
         # INFO: Whitelist for package names
-        if conanfile.name in ["ms-gsl", "cccl"]:
+        if conanfile.name in ["ms-gsl", "cccl", "gnu-config"]:
```
Comment (uilianries) [inline hooks/conan-center.py:609]:
```suggestion
        if conanfile.name in ["ms-gsl", "cccl", "gnu-config", "autoconf", "automake"]:
```

**Comment by uilianries** [comment]
@madebr Done

**Comment by madebr** [comment]
> @madebr Done

Thanks again! 👏

**Comment by uilianries** [comment]
Appveyor failed to fetch the Github repo.

**Comment by danimtb** [comment]
appveyor still red

**Comment by madebr** [comment]
@uilianries The error should have been temporary. I think you can restart the CI.

**Comment by uilianries** [comment]
I have no such power. Let's restart the entire build.

**Comment by madebr** [comment]
@SSE4 
Can we get your final review?

**Review by SSE4 — APPROVED** [review]

**Pull request merged**